app/controllers/home_controller.rb

```diff
@@ -14,8 +14,8 @@ def index
       @city = params["/"]["dataSet"]
       @chart_data = data.pluck(:title, :date_created, :date_updated)
       @count = data.count
-      @created_this_month = data.where(:date_created => 1.month.ago..Time.now).pluck(:title, :category)
-      @updated_this_month = data.where(:date_updated => 1.month.ago..Time.now).pluck(:title, :category)
+      @created_this_month = data.where(:date_created => 1.month.ago..Time.now).pluck(:title, :category, :date_created)
```

Comment:
Adding the date created and the date updated to the information plucked from the datasets created and updated this month.